src/components/VariablesPageEmptyState.vue

```diff
@@ -9,7 +9,7 @@
     </template>
 
     <template #description>
-      Variables store non-sensitive strings.
+      Variables store non-sensitive pieces of JSON.
```

Comment:
```suggestion
      Variables store non-sensitive strings.
```

This is still correct for the existing version of variables.